### PR TITLE
Kinase edition blogpost

### PR DESCRIPTION
**Pull request by t-kimber**
Write a blogpost for the new addition to TeachOpenCADD, the special kinase edition!

### Status
 - [x] Ready to go

### CODE REVIEW

**Comment by t-kimber** [comment]
Dear @AndreaVolkamer and @dominiquesydow,
What do you think about this blogpost for the kinase edition?
:)

File: content/blog/teachopencadd-kinase.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+Kinases are highly conserved in their binding site, which presents a challenge since one ligand may bind not only the designated target (on-target) but also other targets (off-targets), causing mild to severe sides effects. Being able to assess kinase similarity could therefore give insight into potential side-effects.
+
+The proposed pipeline, part of the TeachOpenCADD project, allows the study of kinase similarities from four different angles in an automated and modular fashion.
+
+A short kinase introduction, as well as an overall conclusion, are part of the special kinase edition, establishing 6 news notebooks!
+The analysis of kinase similarity uses information from:
+
+-  sequence,
+-  structure,
```
Comment (dominiquesydow) [inline content/blog/teachopencadd-kinase.md:17]:
```suggestion
-  pocket sequence,
-  pocket structure,
```

Comment (dominiquesydow) [reply content/blog/teachopencadd-kinase.md:17]: _Optional_ addition: Adding the resources we are using (KLIFS, KiSSim, ChEMBL). Could go in parentheses at the end of each bullet point or as a stand-alone sentence.

**Review by dominiquesydow — APPROVED** [review]
Wonderful - concise and informative! Thank you!

File: content/blog/teachopencadd-kinase.md (hunk 1 above)
Comment (t-kimber) [reply content/blog/teachopencadd-kinase.md:17]: Yes, good idea! Will have the resources in parentheses!

File: content/blog/teachopencadd-kinase.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+title: "TeachOpenCADD Kinase edition is out!"
+date: 2021-12-10T00:00:00+02:00
+author: Talia B. Kimber & Dominique Sydow
+---
+
+## Assessing kinase similarity, or how to avoid side effects?
+
+Kinases are highly conserved in their binding site, which presents a challenge since one ligand may bind not only the designated target (on-target) but also other targets (off-targets), causing mild to severe sides effects. Being able to assess kinase similarity could therefore give insight into potential side-effects.
+
+The proposed pipeline, part of the TeachOpenCADD project, allows the study of kinase similarities from four different angles in an automated and modular fashion.
```
Comment (AndreaVolkamer) [inline content/blog/teachopencadd-kinase.md:11]:
```suggestion
The proposed pipeline, part of the [TeachOpenCADD](https://github.com/volkamerlab/teachopencadd) project, allows the study of kinase similarities from four different angles in an automated and modular fashion.
```

File: content/blog/teachopencadd-kinase.md (hunk 1 above)
Comment (AndreaVolkamer) [reply content/blog/teachopencadd-kinase.md]: Good idea, resources including links could be added.

**Review by AndreaVolkamer — APPROVED** [review]
@t-kimber thanks, reads well, very minor suggestion included.

Ready to be merged then.

File: content/blog/teachopencadd-kinase.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+---
+title: "TeachOpenCADD Kinase edition is out!"
+date: 2021-12-10T00:00:00+02:00
+author: Talia B. Kimber & Dominique Sydow
```
Comment (dominiquesydow) [inline content/blog/teachopencadd-kinase.md:4]:
```suggestion
author: Talia B. Kimber
```

Since you wrote all of this, full credit to you :)

**Pull request merged**